packages/cerebral/src/Controller.js

```diff
@@ -24,13 +24,21 @@ class Controller extends EventEmitter {
     this.componentDependencyStore = new DependencyStore()
     this.options = options
     this.flush = this.flush.bind(this)
-    this.devtools = devtools
-    this.model = new Model({}, this.devtools)
+
+    // Only allow devtools in development
+    if (process.env.NODE_ENV !== 'production') {
```

Comment:
I think we need to evaluate if it is necessary to include devtools support in production or not. In the docs devtools are short circuited like this:
`devtools: process.env.NODE_ENV === 'production' ? () => {} : Devtools()`